src/client.js

```diff
@@ -286,34 +285,40 @@ Client.prototype.connect = function(args) {
 	client.save();
 };
 
-Client.prototype.updateToken = function(callback) {
-	var client = this;
-
-	crypto.randomBytes(48, function(err, buf) {
+Client.prototype.generateToken = function(callback) {
```

Comment:
Yeah, I'm kinda with @xPaw on this one, it would essentially just be testing crypto library.